docs/concepts/manifest-builds.md

```diff
@@ -79,11 +79,13 @@ Builds can be performed with different levels of "purity", meaning different lev
 This is controlled with the `sandbox` option.
 
 By default this option is set to `"off"`, which instructs the Flox CLI to perform the build in the root of the repository with no restrictions on network or filesystem access.
-This allows your build to use local caches or intermediate build artifacts that already exist.
+This is convenient because it allows your build scripts to work as they do in your development environment, such as using local caches and intermediate build artifacts that already exist.
+However, there are less guarantees that the build is reproducible, meaning that both the build script and built package may not work the same way on another machine.
```

Comment:
suggestion: 

> However, that also implies that builds can access files or programs specific to your machine, e.g. config files in `$HOME`, applications installed system wide, etc. and potentially embed this information in the resulting binary.
This can subsequently hurt the reproducibility of build script and the ability run binaries on other machines where those referenced files do not exist.